build.gradle

```diff
@@ -31,6 +31,12 @@ dependencies {
 android {
     compileSdkVersion 19
     buildToolsVersion "20.0.0"
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_5
```

Comment:
It is not a "change". If I don't misremember, the amount of warnings that were popped when using 1.7 is annoyingly huge, and most of them are due to complaints of the type "you could do this that is possible since JDK 1.X", where X>5. Obviously we were not going to review the whole codebase to update everything to "Java 7-acceptable" code, because the automated formatting issues you speak about (and that I understand, just didn't really think of it) would be piece of cake when compared to reviewing such a huge, huge set of changes. By setting these options, the IDE stops highlighting everything so you can look at the code without hurting your eyes, and we researched and made sure that there are no compatibility problems at all as long as the specified version is not beyond 1.7.